composer.json

```diff
@@ -18,8 +18,8 @@
   "require-dev": {
     "guzzlehttp/guzzle": "^7",
     "guzzlehttp/psr7": "^2",
-    "openeuropa/code-review": "^2.0.0-alpha6",
-    "phpunit/phpunit": "^10"
+    "openeuropa/code-review": "^2.0",
+    "phpunit/phpunit": "^10.4"
```

Comment:
If we want to use `Generator.php`, it exists in this form since `v10.4`.

Reply:
Lets add a section in composer explaining why we use this specific version, ex https://github.com/openeuropa/oe_whitelabel/blob/1.x/composer.json#L104